### PR TITLE
2016 01 js block on homepage

### PR DESCRIPTION
**Pull request by Johnschmidfront**
@local-girl @MagdaN Don't merge yet. Tell me what you think if you have time. Thanks!

### CODE REVIEW

File: euth_wagtail/templates/base.html [hunk 1]
```diff
@@ -16,6 +16,22 @@
...
+  $(".collapsible_body").hide();
+  //toggle the component with class collapsible_body
+  $(".collapsible_head").click(function()
+  {
+    $(this).next(".collapsible_body").slideToggle(600);
+  });
+});
+</script>
+  <link rel="stylesheet" href="/resources/demos/style.css">
+
```
Comment (MagdaN) [inline euth_wagtail/templates/base.html]: please put your javascript in a seperate javascript file

File: home/templates/home/blocks/info_block.html [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+    <h3><font color="{{ self.font_colour }}">{{ self.heading }}</font></h3>
+
+    {% if self.image.width > 0 %}
+       {% image self.image width-900 class="img-responsive info-img" %}
+    {% else %}
+		{{ self.embedded_video }}
+    {% endif %}
+
+    <p><font color="{{ self.font_colour }}">{{ self.text }}</font></p>
```
Comment (MagdaN) [inline home/templates/home/blocks/info_block.html]: please don't use inline styling, use a classes instead

File: home/templates/home/blocks/info_block.html [hunk 3]
```diff
@@ -0,0 +1,14 @@
+{% load i18n wagtailimages_tags static %}
+<div class="info-div" style="background-color:{{ self.background_colour }}">
```
Comment (MagdaN) [inline home/templates/home/blocks/info_block.html]: please don't use inline styling, use a class instead

**Comment by Johnschmidfront** [comment]
I talked to @local-girl and changed the template and the model - but she's going to do the definitive styling. :)

**Pull request merged**